api/src/main/resources/db/migration/V1.0__create_sсhema.sql

```diff
@@ -18,9 +18,35 @@ create table tag
     constraint unique_tag_name unique (name)
 );
 
-create table certificate_tag
+create table certificate_x_tag
 (
     gift_certificate_id integer not null references gift_certificate on delete cascade,
     tag_id              integer not null references tag on delete cascade,
     primary key (gift_certificate_id, tag_id)
+);
+
+create table consumer
+(
+    id    serial      not null primary key,
+    login varchar(64) not null,
+    constraint unique_login unique (login)
+);
+
+create table purchase
+(
+    id          serial      not null primary key,
+    consumer_id integer     not null references consumer on delete cascade,
+    placed_date timestamptz not null default (now() at time zone 'UTC')
+);
+
+create table gift_certificate_as_purchase_item
```

Comment:
rename to OrderItem. Build relation Many OrderItem s  to One Order (ManyToOne)  or otherwise one Order to many OrderItems (OneToMany) on you choice